ci/vm/provision1.sh

```diff
@@ -13,17 +13,29 @@ install_packages() {
         git \
         libtool &&
       apt-get clean && rm -rf /var/lib/apt/lists/*
+
   elif [[ $OS == *"centos"* ]]; then
-    yum install -y -q centos-release-scl
-    yum install -y -q \
+    # Update repo config to vault links
```

Comment:
maybe we need to only apply vault/sed commands for centos7, so add `if` checking if release is EOL?

Reply:
ok but I meant adding another one under like `if [[ $OS == *"centos7"* ]]; then <sed work> fi`

Reply:
The entire CentOS project is dead now.
In fact, CentOS-7 was the last to reach the EOL.

Reply:
ah i see, centos 8 is already ended too